src/server/routes/unsupportedBrowserRouter.tsx

```diff
@@ -39,6 +39,20 @@ const unsupportedBrowserRouter = (_: Request, res: Response) => {
 
   const jsx = extractor.collectChunks(ReactApp);
   const markup = renderToString(jsx);
+  const analyticsMarkup =
+    `'${process.env.REPORT_ANALYTICS}'`.toLowerCase() !== 'true'
+      ? ''
+      : `<script>
+    if('${process.env.REPORT_ANALYTICS}'.toLowerCase() === 'true'){
+      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+      })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
+
+      ga('create', '${process.env.GOOGLE_ANALYTICS_KEY}', 'auto');
+      ga('send', 'pageview');
+    }
+    </script>`;
```

Comment:
I'm confused why there are two checks of `'${process.env.REPORT_ANALYTICS}'.toLowerCase() === 'true'` here.

I'd also move this script out out of the router function for readability.

```js
const getAnalyticsScript = () =>
  `<script>
    (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
    (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
    m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
    })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
    ga('create', '${process.env.GOOGLE_ANALYTICS_KEY}', 'auto');
    ga('send', 'pageview');
  </script>`;


// in the unsupportedBrowserRouter, inside the html template

${'${process.env.REPORT_ANALYTICS}'.toLowerCase() === 'true' && getAnalyticsScript()}
```